node/node_v2.go

```diff
@@ -61,7 +65,11 @@ func (n *Node) DownloadBundles(ctx context.Context, batch *corev2.Batch, operato
 		relayIndex := rand.Intn(len(cert.RelayKeys))
 		relayKey := cert.RelayKeys[relayIndex]
 		for _, quorum := range cert.BlobHeader.QuorumNumbers {
-			assgn, err := corev2.GetAssignment(operatorState, batch.BlobCertificates[0].BlobHeader.BlobVersion, quorum, n.Config.ID)
+			blobParams, ok := n.BlobVersionParams.Get(cert.BlobHeader.BlobVersion)
```

Comment:
It looks when there is a new version upgrade, it may have an outage since different dispersers/nodes up to the different versions (and potentially cannot reach quorum threshold)?

Reply:
Yes. The blob version needs to be registered onchain well before new blobs start using the new version.

Reply:
1) Add new blob version onchain (multisig transaction)
2) Wait for `OnchainStateRefreshInterval`. If operator nodes, disperser, and relays use the default value (1h), all parts of system will be aware of the new version within this time
3) Blob with new version can be dispersed